src/tests/nativeaot/SmokeTests/PInvoke/PInvoke.cs

```diff
@@ -1019,6 +1029,48 @@ public static void TestComInteropNullPointers()
             var result = IsNULL(comPointer);
             ThrowIfNotEquals(true, IsNULL(comPointer), "COM interface marshalling null check failed");
         }
+
+        public static void TestComInteropRegistrationReqired()
+        {
+            Console.WriteLine("Testing COM Interop registration process");
+            ComObject target = new ComObject();
+            try
+            {
+                CaptureComPointer(target);
+                throw new Exception("Cannot work without ComWrappers.RegisterForMarshalling called");
+            }
+            catch (InvalidOperationException)
+            {
+            }
+        }
+
+        public static void TestComInteropReleaseProcess()
+        {
+            Console.WriteLine("Testing CCW release process");
+            ComWrappers wrapper = new SimpleComWrapper();
+            ComWrappers.RegisterForMarshalling(wrapper);
+            WeakReference comPointerHolder = CreateComReference();
+
+            GC.Collect();
+            ThrowIfNotEquals(true, comPointerHolder.IsAlive, ".NET object should be alive");
+
+            ReleaseComPointer();
+
+            GC.Collect();
+
+            ThrowIfNotEquals(false, comPointerHolder.IsAlive, ".NET object should be disposed by then");
```

Comment:
```suggestion
            GC.Collect();
            ThrowIfNotEquals(false, comPointerHolder.IsAlive, ".NET object should be disposed by then");
```
Nit: to match the formatting of the block above